templates/zerver/help/automatic-theme-detection.md

```diff
@@ -0,0 +1,17 @@
+# Automatic Theme Detection
+
+By default, Zulip detects your system's screen mode automatically
+and is adjusted accordingly.
+
+### Disable Automatic Theme Detection
+
+{start_tabs}
+
+{settings_tab|display-settings}
+
+2. Under **Display settings**, uncheck the option **Detect browser's theme automatically**.
+
+Your system's preference will remain and then you can change it from the **Night Mode** setting
+below.
+
+{end_tabs}
```

Comment:
I'd prefer that we just make this a section in the existing "Night theme" article, since I think that's where folks will look for information on the idea?

Reply:
Shall we remove the ```automatic-theme-detection.md``` section entirely or just move the mentioned lines in the ```night-mode.md``` section?